src/lib/investor/investor-foxy/api/api.ts

```diff
@@ -15,7 +15,7 @@ import { ethers } from 'ethers'
 import { toLower } from 'lodash'
 import { getAddress } from 'viem'
 import { bn, bnOrZero } from 'lib/bignumber/bignumber'
-import { MAX_ALLOWANCE } from 'lib/investor/constants'
+import { MAX_ALLOWANCE } from 'lib/investor/constants/allowance'
```

Comment:
```suggestion
import { maxUint256 } from 'viem'
```

Reply:
:crazyeyes2:

Reply:
still applicable to use viem here and then delete `lib/investor/constants`